modules/tomcat/manifests/params.pp

```diff
@@ -2,29 +2,31 @@
 # Class: tomcat::params  related to conf/*.* file of Apache Tomcat and httpd
 #
 
-class tomcat::params
-{
+class tomcat::params (
+# config variables for aplication
+$man_user		      = "$profile::tomcat::man_user",	
+$password             = "$profile::tomcat::password", 
+$host_name            = "$profile::tomcat::host_name",
+$docBase              = "$profile::tomcat::docBase", 
+){
 
 # config variables for tomcat
-$user             = 'tomcat'
-$password         = 'tomcat' 
-$group            = 'tomcat'
-$java_home        = '/usr/java/jdk1.8.0_162/jre'
-$catalina_home    = '/usr/share/tomcat'
-$catalina_base    = '/var/lib/tomcat/webapps' 
-$confdir          = '/etc/tomcat'
-$packages         = ['tomcat', 'tomcat-webapps', 'tomcat-admin-webapps']
+$user 				  = 'tomcat'
```

Comment:
but other are hard-coded in module code?
Move them to parameters section as well.